.github/workflows/release.yml

```diff
@@ -200,11 +200,23 @@ jobs:
       base64-subjects: "${{ needs.sbom-assests.outputs.hashes }}"
       provenance-name: "karmada-sbom.intoto.jsonl"
       upload-assets: true
+
   update-krew-index:
-    needs: release-assests
+    needs: 
+    - release-assests
     name: Update krew-index
     runs-on: ubuntu-22.04
     steps:
+    - name: check latest tag
+      id: check-latest
+      run: |
+        # get the latest version
+        export STABLE_TAG=`gh api repos/karmada-io/karmada/releases/latest | jq -r '.tag_name'`
+        echo "Got the stabe tag:$STABLE_TAG"
+        echo "event.tag:"${{ github.event.release.tag_name }}
+        echo "stableTag=$STABLE_TAG" >> "$GITHUB_OUTPUT"
     - uses: actions/checkout@v4
+      if: steps.check-latest.outputs.stableTag == github.event.release.tag_name
```

Comment:
Maybe we don't need to check out the code.

Reply:
it's not true,  krew-release-bot need the file of https://github.com/karmada-io/karmada/blob/master/.krew.yaml , https://github.com/rajatjindal/krew-release-bot?tab=readme-ov-file#basic-setup

Reply:
OK, then keep it.